src/App/Pages/TabsPage.cs

```diff
@@ -78,12 +82,26 @@ public TabsPage(AppOptions appOptions = null, PreviousPageInfo previousPage = nu
         protected override async void OnAppearing()
         {
             base.OnAppearing();
+            _broadcasterService.Subscribe(nameof(TabsPage), async (message) =>
+            {
+                if (message.Command == "syncCompleted")
+                {
+                    Device.BeginInvokeOnMainThread(async () => await UpdateVaultButtonTitleAsync());
+                }
```

Comment:
Add try...catch, if an exception occurs there it will crash the app given that this is similar to an async void method. Or at least try catch inside `UpdateVaultButtonTitleAsync`